controllers/vaultpkisecret_controller.go

```diff
@@ -292,25 +309,26 @@ func (r *VaultPKISecretReconciler) handleDeletion(ctx context.Context, l logr.Lo
 	return nil
 }
 
-func (r *VaultPKISecretReconciler) addFinalizer(ctx context.Context, l logr.Logger, s *secretsv1beta1.VaultPKISecret) error {
+func (r *VaultPKISecretReconciler) addFinalizer(ctx context.Context, s *secretsv1beta1.VaultPKISecret) (bool, error) {
```

Comment:
Is the `bool` return being used anywhere?

Reply:
Good catch, updated in d7a5ad7c97a73d39d8f253fca0ae8c3f4965d8d0